include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -2177,6 +2177,21 @@ namespace internal
                           vv,
                           my_data.jacobians[0][offset + q][d][e]);
                     }
+                  // Also store the jacobian
+                  // Flip columns instead so that it matches if you call
+                  // invert(jacobians[q]) for general faces.
+                  if (face_type[face] <= affine)
+                    for (unsigned int e = 0; e < dim; ++e)
+                      {
+                        const unsigned int ee =
+                          ExtractFaceHelper::reorder_face_derivative_indices<
+                            dim>(interior_face_no, e);
```

Comment:
Well it is. I wasn't sure which one of the two that was being used. Looking at it again makes it clear that I don't need this part, but only the same thing I did a couple of lines above. Since this is not used by the piola transform, and not anywhere else, I see no point in keeping it. I can keep it though, for consistency, and just remove the `reorder_face_derivative_indices` since it essentially does not do anything.

Reply:
Or I mean it does `Assert(index < dim, ExcInternalError());` so I guess it has some value.

Reply:
It is actually only used in 2 out of 12 places with reference-cell passed in. And what I said above is not correct. Sorry